x/ref/runtime/internal/cloudvm/aws.go

```diff
@@ -58,63 +59,86 @@ const (
 var (
 	onceAWS sync.Once
 	onAWS   bool
+	imdsv2  bool
```

Comment:
Maybe rename this to `onImdsv2` or something different than imdvs2 since local functions override the scope and is a little confusing

Reply:
done!